README.md

```diff
@@ -1,3 +1,47 @@
-# nodejs-boilerplate
+[![commercetools logo][commercetools-icon]][commercetools]
 
-## In active development
+# Node.js boilerplate
+[![Travis Build Status][travis-icon]][travis]
+[![Codecov Coverage Status][codecov-icon]][codecov]
+[![David Dependencies Status][david-icon]][david]
+[![David devDependencies Status][david-dev-icon]][david-dev]
+
+Start your next [Node.js commercetools](http://commercetools.github.io/nodejs-tools/) project in seconds.
+
+## Features
+- Babel: use new JavaScript features while keeping compatibility.
+- TravisCI: config and setup included.
+- ESLint: config to keep the coding style consistent.
+- Templates: includes readme and package.json templates.
+- Tests: uses [tape][tape] for testing.
+
+## Requirements
+Make sure you have installed all of the following prerequisites on your development machine:
+  * Git - [Download & Install Git](https://git-scm.com/downloads). MacOS and Linux machines typically have this already installed.
+  * Node.js - [Download & Install Node.js](https://nodejs.org/en/download/) and the npm package manager. Make sure to get the latest active LTS version. You could also use a Node.js version manager such as [n](https://github.com/tj/n) or [nvm](https://github.com/creationix/nvm).
+
+## Usage
+  1. Clone this repository:
+
+  ```bash
+  git clone --depth=1 https://github.com/commercetools/nodejs-boilerplate.git
+  ```
+  2. Clean up, setup and install.  
+  Takes the project name and an optional second directory like `bin` to use:
```

Comment:
"You can pass the project name and an additional directory name that you want to use, for example 'bin' "